tests/test_scraper.py

```diff
@@ -52,9 +52,13 @@ def test_blast_scraping(scraper, testdata_prefix, caplog):
   caplog.set_level(logging.DEBUG)
   scraper.scrape_blast(type='seq_type',file_list=["{}/blast_single_loci.txt".format(testdata_prefix)])
   assert "candidate" in caplog.text
+
   caplog.clear()
-  scraper.scrape_blast(type='resistance',file_list=["{}/blast_single_resistance.txt".format(testdata_prefix)])
-  assert "candidate" in caplog.text
+  hits = scraper.scrape_blast(type='resistance',file_list=["{}/blast_single_resistance.txt".format(testdata_prefix)])
```

Comment:
```suggestion
  hits = scraper.scrape_blast(type='resistance',file_list=[f"{testdata_prefix}/blast_single_resistance.txt"])
```

Reply:
I wonder if we should perhaps keep the formatting code as is for consistency, and change everything at once later if needed instead?